var/spack/repos/builtin/packages/py-poxy/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPoxy(PythonPackage):
+    """Documentation generator for C++"""
+
+    homepage = "https://github.com/marzer/poxy"
+    pypi = "poxy/poxy-0.18.0.tar.gz"
+
+    license("MIT", checked_by="pranav-sivaraman")
+
+    version("0.18.0", sha256="f5da8ff04ec08859bfd1c8ec6ef61b70e3af630915a4cce6a3e377eec3bcd3d4")
+
+    depends_on("py-setuptools", type="build")
+
+    with default_args(type=("build", "run")):
+        depends_on("python@3.7:")
+        depends_on("py-misk")
```

Comment:
Should this be `depends_on("py-misk@0.8.1:")`?

From https://github.com/marzer/poxy/blob/main/pyproject.toml:
```
...
dependencies = [
    'misk >= 0.8.1',
    'bs4',
    'jinja2',
    'pygments',
    'html5lib',
    'lxml',
    'tomli',
    'schema != 0.7.5',
    'requests',
    'trieregex',
    'colorama',
]
...
```

Reply:
Yes. Technically `py-misk@0.8.1` is the only that exists in the builtin repo, but I'll add make the change.

Reply:
People have come along later and added older versions to packages so being explicit is generally safer.